source/admin_manual/list_of_events.rst

```diff
@@ -165,6 +165,247 @@ Global Fields
+| real_remote_ip    | Real remote IP as seen by the server                       |
++-------------------+------------------------------------------------------------+
+| real_local_port   | Real local port as seen by the server                      |
++-------------------+------------------------------------------------------------+
+| real_remote_port  | Real remote port as seen by the server                     |
++-------------------+------------------------------------------------------------+
+| error             | Error string if error occured                              |
++-------------------+------------------------------------------------------------+
+
+
+auth_client_request_started
+---------------------------
+
++--------------------+-----------------------------------------------------------+
+| Field             | Description                                                |
++===================+============================================================+
+| service           | Service name, such as imap3, pop3, lmtp                    |
++-------------------+------------------------------------------------------------+
+| session           | Session identifier                                         |
++-------------------+------------------------------------------------------------+
+| local_name        | TLS SNI                                                    |
++-------------------+------------------------------------------------------------+
+| local_ip          | Local IP client connected to                               |
++-------------------+------------------------------------------------------------+
+| remote_ip         | Remote IP of client                                        |
++-------------------+------------------------------------------------------------+
+| local_port        | Local port client connected to                             |
++-------------------+------------------------------------------------------------+
+| remote_port       | Remote port of client                                      |
++-------------------+------------------------------------------------------------+
+| real_local_ip     | Real local IP as seen by the server                        |
++-------------------+------------------------------------------------------------+
+| real_remote_ip    | Real remote IP as seen by the server                       |
++-------------------+------------------------------------------------------------+
+| real_local_port   | Real local port as seen by the server                      |
++-------------------+------------------------------------------------------------+
+| real_remote_port  | Real remote port as seen by the server                     |
++-------------------+------------------------------------------------------------+
+
+auth_client_request_challenged
+------------------------------
+
+*same as auth_client_request_started*
```

Comment:
For consistency use same as for e.g. dict_server_lookup_finished:

Event emitted when dict server finishes lookup. Same fields as dict_lookup_finished.

Where dict_lookup_finished is a ref.

Reply:
better now?